plugins/network-generator/src/main/kotlin/dev/icerock/moko/network/MultiPlatformNetworkGeneratorPlugin.kt

```diff
@@ -46,6 +46,7 @@ class MultiPlatformNetworkGeneratorPlugin : Plugin<Project> {
 
                     inputSpec.set(spec.inputSpec?.path)
                     packageName.set(spec.packageName)
+                    PathOperationsFilter.addAllTagsToFilter(spec.filterTags)
```

Comment:
it's will be global filter...not one spec. so we should move it from spec block. or implement correct multiple specs filters

Reply:
To fix this, I added filter initialization in the `doFirst` block and cleanup singleton in the `doLast` block of the tasks.

Reply:
Remade passing filter tags through additional properties.